### PR TITLE
meal sharing endpoint

### PR DESCRIPTION
**Pull request by Atefeh-Behroozi**
Finished my homework

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -21,59 +21,59 @@ app.use(cors());
...
+     const meal = await knex.raw("SELECT * FROM meal ORDER BY id");
+     res.json(200).json(meal[0]);
+   } catch (error) {
+    res.status(404).json(error);
+   }
+ });
+ 
+ app.get('/first-meal', async (req, res) => {
+   try {
+     const meal = await knex.raw("SELECT * FROM meal ORDER BY id LIMIT 1");
```
Comment (raihanmehran) [inline src/backend/app.js]: would be better to add ASC keyword to the query

**Review by raihanmehran — COMMENTED** [review]
Well done @Atefeh-Behroozi !
just a tiny suggestion, it would be better to name the variables in plural, when you returning a list or array of objects.

Happy Coding :)

File: src/backend/app.js (hunk 1 above)
Comment (Atefeh-Behroozi) [reply src/backend/app.js]: Ok, thank you for your review. I fix it now :)

**Comment by Atefeh-Behroozi** [comment]
Thank you so much for the feedback. I've change name the variables to plural and add ASC keyword to the query :)

**Pull request merged**